File: 2 - Condições/condicoes12.rb
#Brincadeira do ímpar ou par: escreva um programa que leia dois nomes e dois valores inteiros, que correspondem ao que cada um colocou, e informe quem ganhou o "impar ou par".

puts "Digite o nome do primeiro jogador:"
nome1 = gets.chomp
puts "Digite o nome do segundo jogador:"
nome2 = gets.chomp

puts "Digite o número de dedos do primeiro jogador:"
mao1 = gets.to_i
puts "Digite o número de dedos do segundo jogador:"
mao2 = gets.to_i

if( (mao1 + mao2) % 2 == 1)
	puts "O jogador vencedor é: #{nome1}"
else
	puts "O jogador vencedor é: #{nome2}"
end

File: 2 - Condições/condicoes12.c
//Brincadeira do ímpar ou par: escreva um programa que leia dois nomes e dois valores inteiros, que correspondem ao que cada um colocou, e informe quem ganhou o "impar ou par".

#include <stdio.h>

int main() {
  char nome1[4];
  char nome2[4];
  int mao1;
  int mao2;
  printf("Digite o nome do primeiro jogador: \n");
  gets(nome1);
  printf("Digite o nome do segundo jogador: \n");
  gets(nome2);
  printf("Digite o numero de dedos do primeiro jogador: \n");
  scanf("%d", &mao1);
  printf("Digite o numero de dedos do segundo jogador: \n");
  scanf("%d", &mao2);

  if ((mao1 + mao2) & 1) {
    printf("Vencedor: %s\n", nome1);
  }
  else {
    printf("Vencedor: %s\n", nome2);
  }

  return 0;
}
